### ISSUE TITLE
[Roadmap] 2019 - Q2

### ISSUE THREAD

**Issue by newmuis** [ampproject/wg-stories#10]
Stories Working Group's focus for Q2 of 2019 is making existing features more robust, as well as exploring new avenues for stories (interactivity, live stories, and reactions)

- [ ] Media playback reliability
- [ ] Support better prerendering for stories
- [ ] Support tapping to next story from last page when embedded in an AMP viewer
- [ ] Stories-specific templating API
- [ ] Deprecate 0.1
- [ ] Support amp-live-list for live stories
- [ ] Start brainstorming for reactions in stories
- [ ] Finish embedded components
- [ ] Launch branching

**Comment by swissspidy** [comment]
> Support tapping to next story from last page when embedded in an AMP viewer

I've seen many people try to swipe through stories, much like you can do on Instagram or Snapchat. Is this something that is considered for this feature? e.g. tap = go to next page, swipe = go to next story?

**Comment by gmajoulet** [comment]
Swiping to the next story is a feature that's already implemented, in the context of a Viewer. This is why it's not mentioned here.
If you look at the "Top Stories" carousel on a Search Results Page, you can swipe from one AMP article to another. AMP Stories will offer the exact same behavior, as soon as the new integrations announced at I/O are launched.
On top of that, in the same Viewer context, we want to make tapping on the last page navigate to the next story.

TLDR; tap = next page, tap on last page = next story, swipe = next story

**Comment by newmuis** [comment]
Closed in favor of #14, the Q3 roadmap update.
